meson.build

```diff
@@ -1,6 +1,6 @@
 project('rpminspect-data-fedora',
-        version : '1.1',
-        license : 'GPLv3+',
+        version : '4.0',
```

Comment:
The version should still be 1.1 because that's the version of rpminspect-data-fedora, not the license version.